objax/gradient.py

```diff
@@ -71,3 +71,28 @@ def __call__(self, *args):
                                        list(args))
         self.vc.subset(BaseState).assign(changes)
         return g, outputs
+
+
+class Grad(ModuleWrapper):
```

Comment:
Can you move the new function below GradValues (we try to keep the order alphabetical within files)?

Reply:
Sure. This applies to the test script also right?